syntax = "proto3";

package communication_objects;

message ActionSpaceProto {
    int32 discrete = 1;
    int32 param = 2;
    int32 acs_space = 3;
}

message EnvSpecsProto {
    int32 observation_space = 1;
    ActionSpaceProto action_space = 2;
    int32 num_agents = 3;
}

message MultiEnvSpecsProto {
    EnvSpecsProto env_specs = 1;
    int32 num_envs = 2;
}

message LearnerSpecsProto {
    string data = 1;
}
